File: ClientApp/src/app/models/reservation.ts
export class Reservation {
  ReservationDate: Date;
  Email: string;
  OwnerName: string;
  ClassName: string;
  NumberPersons: number;


  constructor(ReservationDate?: Date, Email?: string, OwnerName?: string,ClassName?:string,NumberPersons?:number) {
    this.ReservationDate = ReservationDate;
    this.Email = Email;
    this.OwnerName = OwnerName;
    this.ClassName = ClassName;
    this.NumberPersons = NumberPersons;
   



  }

}

File: Models/Reservation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JUSTMOVE.Models
{
    public class Reservation
    {
        public string Email { get; set; }
        public string OwnerName { get; set; }
        public string ClassName { get; set; }
        public DateTime ReservationDate { get; set; }
       
        public int NumberPersons { get; set; }
       

    }
}
